components/ReferenceManager/references/reference_uploader/ReferenceManualUploadDrawer.tsx

```diff
@@ -80,6 +82,14 @@ export default function ReferenceManualUploadDrawer(): ReactElement {
     setReferenceSchemaValueSet,
   });
 
+  useEffect(() => {
+    const rootLeftSidebarWidth =
+      document.querySelector(".root-left-sidebar")?.clientWidth || 0;
+    const refManagerNavbarWidth =
+      document.querySelector(".ToggleableNavbarLeft")?.clientWidth || 0;
+    setCalculatedLeftMargin(rootLeftSidebarWidth + refManagerNavbarWidth + 2);
```

Comment:
Will this cause any flashing? If we server side render some pieces of this, and then grab things from client side, typically the app will flash

Reply:
Is it better here to set a constant width for some of these classes so that we don't have to set this in javascript?

Reply:
It should not cause flashing because it executes within a `useEffect` which only fires on the client side.

Initially, these values were hard coded but I couldn't rely on the hard-coded component widths because some components will not be visible in certain resolutions. For example, the left-most sidebar is not visible in mobile screens. The ref. manager sidebar turns into a drawer at 900 px, etc...

Needed a more accurate solution.